src/networkrequest.h

```diff
@@ -46,7 +50,9 @@ class NetworkRequest final : public QObject {
 
   QNetworkRequest& requestInternal() { return m_request; }
 
-  void auth(const QByteArray& authorizationHeader);
+  static const QByteArray authorizationHeader();
+
+  void auth(const QByteArray& authorizationHeader = "");
```

Comment:
This is old code, but should this have been named setAuthHeader() or setAuthorizationHeader()? Otherwise it is confusing at the call site.